### ISSUE TITLE
Cleanup after the alpha release

### ISSUE THREAD

**Issue by adriatic** [monterey-framework/monterey#348]
I would like to get rid of the [global issues](https://github.com/monterey-framework/global-issues) repository as it has lost its relevance (if it ever had some) and its current presence may create confusion.

We should also comb through this issues article and close all that do not matter after the alpha release. I know that @JeroenVinke has been doing this throughout the Monterey existence, so perhaps we can to this even more energetically now.

**Comment by JeroenVinke** [comment]
Agreed with removing https://github.com/monterey-framework/global-issues

> We should also comb through this issues article and close all that do not matter after the alpha release. 

Always in favor of this. I propose to take all idea's (that we do not intend to implement right away) and add them to https://aurelia-ui-toolkits.gitbooks.io/monterey-user-manual/content/introduction/montereyfuture_evolution.html. If we organize this article a bit better then we can describe each idea. When we decide to start working on one of these features we can create issues for design, discussion and implementation. Consider the article as our product backlog (things we would like to implement one day) and our issues in the github repository as our sprint backlog.

I think that using the following phases is very logical: discussion (design) -> implementation -> bugfix/improvement. We should always be clear about what needs to be done in order to close an issue. https://github.com/monterey-framework/monterey/issues/343 is an excellent issue, because it is very clear about what needs to be done.

Let me know what you think

**Comment by JeroenVinke** [comment]
https://github.com/monterey-framework/monterey/issues/82 is something we could add to our product backlog, as it is something we wish to do one day, but not right away

**Comment by adriatic** [comment]
> https://github.com/monterey-framework/monterey/issues/82 is something we could add to our product backlog

Yes, I agree.

**Comment by adriatic** [comment]
Re: https://github.com/monterey-framework/monterey/issues/348#issuecomment-249436332

As the first step leading to the solution you suggested, I am writing [Moving to the next release](https://github.com/monterey-framework/monterey/issues/349), which (article) I will divide into categories, comment each entry of each category, so we can discuss it further.
